spec/mongoid/criteria/queryable/extensions/symbol_spec.rb

```diff
@@ -13,6 +13,10 @@
       end
     end
 
+    after do
+      Symbol.undef_method(:fubar)
```

Comment:
This spec dynamically monkey-patches Symbol, so I need to clean it up after the spec runs.